app/shared/productTypes.tsx

```diff
@@ -194,8 +195,15 @@ export const ProductTypes: { [productKey: string]: ProductType } = {
     productPage: {
       title: "Membership",
       navLink: navLinks.membership,
-      noProductInTabCopy:
-        "To manage your existing contribution or subscription, please select from the tabs above.",
+      noProductInTabCopy: (
```

Comment:
These could be wrapped in a function which takes the two other tabs as params to avoid repetition of the copy and JSX.